src/js/components/Form/stories/AggregateValidation.js

```diff
@@ -19,12 +19,12 @@ export const AggregateValidation = () => {
           onChange={(nextValue) => setValue(nextValue)}
           onSubmit={({ value: nextValue }) => console.log(nextValue)}
         >
-          <FormField label="Name" name="name" required>
-            <TextInput name="name" type="name" />
+          <FormField htmlFor="name" label="name" name="name" required>
+            <TextInput id="name" name="name" type="name" />
```

Comment:
Can we just change this so that the input has an aria-label?
```suggestion
          <FormField label="Name" name="name" required>
            <TextInput aria-label="name" name="name" type="name" />
```